core/src/main/java/de/danoeh/antennapod/core/feed/FeedMedia.java

```diff
@@ -168,10 +167,6 @@ public MediaType getMediaType() {
     }
 
     public void updateFromOther(FeedMedia other) {
-        // reset to new if feed item did link to a file before
-        if(TextUtils.isEmpty(download_url) && !TextUtils.isEmpty(other.download_url)) {
```

Comment:
The feed would have to include the enclosure information with empty href.
If that happens, I would really like to scream at the moron who managed to do this ;)